docsteady/__init__.py

```diff
@@ -101,6 +101,10 @@ def generate_spec(format, username, password, folder, path):
     requirements_to_testcases = OrderedDict(sorted(Config.REQUIREMENTS_TO_TESTCASES.items(),
                                                    key=lambda item: alphanum_key(item[0])))
 
+    for testcase in Config.CACHED_LIBTESTCASES.keys():
+        print(testcase)
```

Comment:
Is this for debugging?

Reply:
Yes, I am removing it... and other commented debugs.